services/property-svc/internal/property/aggregate/aggregate.go

```diff
@@ -40,76 +40,94 @@ func LoadPropertyAggregate(ctx context.Context, as hwes.AggregateStore, id uuid.
 }
 
 func (a *PropertyAggregate) initEventListeners() {
-	a.RegisterEventListener(propertyEventsV1.PropertyCreated, a.onPropertyCreated)
-	a.RegisterEventListener(propertyEventsV1.PropertyDescriptionUpdated, a.onDescriptionUpdated)
-	a.RegisterEventListener(propertyEventsV1.PropertySetIDUpdated, a.onSetIDUpdated)
-	a.RegisterEventListener(propertyEventsV1.PropertySubjectTypeUpdated, a.onSubjectTypeUpdated)
-	a.RegisterEventListener(propertyEventsV1.PropertyNameUpdated, a.onNameUpdated)
-	a.RegisterEventListener(propertyEventsV1.PropertyFieldTypeDataCreated, a.onFieldTypeDataCreated)
-	a.RegisterEventListener(propertyEventsV1.PropertyFieldTypeDataAllowFreetextUpdated, a.onAllowFreetextUpdated)
-	a.RegisterEventListener(propertyEventsV1.PropertyFieldTypeDataSelectOptionsUpserted, a.onFieldTypeDataSelectOptionsUpserted)
-	a.RegisterEventListener(propertyEventsV1.PropertyFieldTypeDataSelectOptionsRemoved, a.onFieldTypeDataSelectOptionsRemoved)
-	a.RegisterEventListener(propertyEventsV1.PropertyArchived, a.onPropertyArchived)
-	a.RegisterEventListener(propertyEventsV1.PropertyRetrieved, a.onPropertyRetrieved)
+	a.RegisterEventListener(hwes.MessageToEventName(&pbTechnicalEventsV1.PropertyCreatedEvent{}), a.onPropertyCreated)
```

Comment:
we could move this into RegisterEventListener and get rid of a lot of code duplication + get better type guarantees